.static/detekt-config.yml

```diff
@@ -0,0 +1,172 @@
+build:
+  maxIssues: 0
+
+console-reports:
+  exclude:
+    - "ProjectStatisticsReport"
+    - "ComplexityReport"
+    - "NotificationReport"
+
+complexity:
+  LongParameterList:
+    active: false
+  LargeClass:
+    excludes: "**/test/**,**/*.Test.kt"
+  ReplaceSafeCallChainWithRun:
+    active: true
+
+formatting:
+  android: true
+  ImportOrdering:
+    active: true
+    autoCorrect: true
+  AnnotationOnSeparateLine:
+    active: true
+    autoCorrect: true
+  EnumEntryNameCase:
+    active: true
+    autoCorrect: true
+  AnnotationSpacing:
+    active: true
+    autoCorrect: true
+  ArgumentListWrapping:
+    active: true
+    autoCorrect: true
+
+naming:
+  ConstructorParameterNaming:
+    privateParameterPattern: "(_)?[a-z][A-Za-z0-9]*"
+  TopLevelPropertyNaming:
+    active: false
+  VariableNaming:
+    active: true
+    variablePattern: "[a-z][A-Za-z0-9]*"
+    privateVariablePattern: "(_)?[a-z][A-Za-z0-9]*"
+    excludeClassPattern: "$^"
+  NonBooleanPropertyPrefixedWithIs:
+    active: true
+    excludes: "**/test/**,**/*.Test.kt"
+  FunctionNaming:
+    ignoreAnnotated: ["Benchmark"]
+
+performance:
+  SpreadOperator:
+    active: false
```

Comment:
Just saw a potential fix for this https://github.com/detekt/detekt/releases/tag/v1.14.2 so maybe it doesn't need to be disabled?

Reply:
This check is not broken, it's just, personally, I don't know what I would do about it. I like spreading arrays and I'm willing to pay the price.

In the PR branch, for instance, it complains about this snippet (`ModelAst.kt:135`):
```kotlin
    private fun elementDeclaredType(type: TypeElement, astTypes: List<AstType>) =
        types.getDeclaredType(
            type,
            *astTypes.map { // this line
                (it as ModelAstType).type
            }.toTypedArray()
        )
```

Reply:
Ah yep, the linked fixes doesn't actually solve it.